src/components/App/index.vue

```diff
@@ -175,6 +175,10 @@ export default {
   },
   sockets: {
     error(error) {
+      // these are handled internally and shouldn't be forwarded to Sentry
+      if (error.message === "xhr poll error") {
```

Comment:
After checking out the [socket events](https://socket.io/docs/client-connection-lifecycle/#Events), we might be able to remove this listener since it looks like it's not a valid socket event. However, I'm not sure if `xhr poll error` appears in both `connect_error` and `reconnect_error` or only one of them

Reply:
that's the docs for 3.0, we're on 2.3.0

The code that lines up with that release can be explored here: https://github.com/socketio/socket.io/tree/47161a65d40c2587535de750ac4c7d448e5842ba/lib

and there's definitely and `error` event

https://socket.io/docs/v2/emit-cheatsheet/

Of course there's _also_ `connect_error` and `reconnect_error`

it looks like they drop several reserved event types in v3.

Reply:
might be that the best way to test this is to have it console.log the error from all three with an identifier and try to open up a session locally and see if we can figure out which one is logging it

Reply:
Nice catch and that sounds like a plan!